dask/array/backends.py

```diff
@@ -71,3 +90,20 @@ def _concatenate(L, axis=0):
             raise ValueError(msg)
 
     concatenate_lookup.register(scipy.sparse.spmatrix, _concatenate)
+
+    def _tensordot(a, b, axes):
```

Comment:
It seems like this implementation is exactly the same as the one above, is that correct? I would suggest we have the implementation on a separate function and have both lookups calling that function, this will simplify future maintenance.

Reply:
That is correct. I will fix it to use the same implementation for CuPy sparse and SciPy sparse.